<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_main">
    <item>
     <layout class="QHBoxLayout" name="startModulesLayout">
      <item>
       <widget class="QPushButton" name="startModulesButton">
        <property name="text">
         <string>Start Modules</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopApplicationButton">
        <property name="text">
         <string>Stop Application</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearButton">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="stopModulesLayout">
      <item>
       <widget class="QPushButton" name="stopModule1Button">
        <property name="text">
         <string>Stop Module 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopModule2Button">
        <property name="text">
         <string>Stop Module 2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopModule3Button">
        <property name="text">
         <string>Stop Module 3</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_main">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>6</number>
      </property>
      <widget class="QWidget" name="plotWidget">
       <layout class="QVBoxLayout" name="plotLayout">
        <item>
         <widget class="QStackedWidget" name="plotStackedModeWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <widget class="QWidget" name="page2DPlot"/>
          <widget class="QWidget" name="page3DPlot"/>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="plotModeWidget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>50</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QRadioButton" name="rB3DPlot">
             <property name="text">
              <string>3DPlot</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rB2DPlot">
             <property name="text">
              <string>2DPlot</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPlainTextEdit" name="logTextEdit">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>300</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionSettings"/>
   </widget>
   <addaction name="menuSettings"/>
  </widget>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <addaction name="actionSettings"/>
 </widget>
 <resources/>
 <connections/>
</ui>
